components/messaging/CurrentConversation.tsx

```diff
@@ -41,7 +32,7 @@ const CurrentConversation: React.FC = () => {
     };
 
     fetchMessagesForCurrentConversation();
-  }, [currentConversation]);
+  }, [currentConversation, setCurrentMessages]);
 
```

Comment:
Not sure we need this new dependency either, we don't need all the messages fetched when  setcurrentMessages runs, that's what the supabase reatime is managing.

Reply:
Marking this as resolved following our call :)